packages/config/config/devices/0x015a/jtb-zmnkgd1.json

```diff
@@ -0,0 +1,16 @@
+{
+	"manufacturer": "Jin Tao Bao",
+	"manufacturerId": "0x015a",
+	"label": "JTB-ZMNKGD1",
```

Comment:
@QubinoHelp I assume this is the manufacturer that produced this device for you?
We typically put the brand name in the `manufacturer` field, so should this be changed to Qubino, or is this one of those white-label devices that's sold under multiple brands?

Reply:
Yes that is a re brand how we call it, the actual producers are them. It can be added under multiple brands I suppose.

Reply:
Ok then we'll have to leave this as-is.